### ISSUE TITLE
Add optionals for new iOS view bindings

### ISSUE THREAD

**Issue by Alex009** [icerockdev/moko-mvvm#40]
with new ios view bindings required write
```
let errorText: LiveData<StringDesc> = viewModel.state.error().map { $0 as? StringDesc ?? StringDesc.Raw(string: "") } as! LiveData<StringDesc>
errorLabel.bindText(liveData: errorText)
```
because optionals not supported

**Comment by Alex009** [comment]
i think in this issue we should try remove requirement of naming `MultiPlatformLibrary` for kotlin framework. 
to do it i suggest:
revert action of binding extensions. now we add extension function to `UIKit` classes and Kotlin/Native can't add it as extension...but when we add extension of kotlin class - it exported correct.
so i suggest to change:
```kotlin
fun UIButton.bindEnabled(
    liveData: LiveData<Boolean>,
    enabledColor: UIColor? = null,
    disabledColor: UIColor? = null
) {
    ...
}
```
to
```kotlin
fun LiveData<Boolean>.bindEnabled(
    button: UIButton,
    enabledColor: UIColor? = null,
    disabledColor: UIColor? = null
) {
    ...
}
```

**Comment by Dorofeev** [comment]
In version 0.9.0 new iOS bindings still have same problem. For example [this](https://github.com/icerockdev/moko-mvvm/blob/00e1407737b2abf8ebe9171bba686f28cc3523b8/mvvm-livedata/src/iosMain/kotlin/dev/icerock/moko/mvvm/livedata/UILabelBindings.kt) bindings not support optional values

**Comment by Alex009** [comment]
implemented in #100 and released in 0.9.1
